src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -390,6 +390,7 @@ private void PackageManagerLoaded(object sender, RoutedEventArgs e)
 
         private void PackageManagerUnloaded(object sender, RoutedEventArgs e)
         {
+            Model =  null;
```

Comment:
Same thing applies here, isn't the control the only reference to this data?
Is the control not getting cleaned up?

Reply:
I just followed "Retention path" from memory dump. I am sure there are other problems with XAML code code. But I have very basic experience with XMAL, can't comment what would be other issues.